File: --Legacy/--Docs/update-getting-started.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-

import sqlite3
import os

# Get the directory of this script
script_dir = os.path.dirname(os.path.abspath(__file__))
db_path = os.path.join(script_dir, 'database.sqlite')

# Read content files
content_en_path = os.path.join(script_dir, 'content', 'getting-started-en.html')
content_ar_path = os.path.join(script_dir, 'content', 'getting-started-ar.html')

try:
    with open(content_en_path, 'r', encoding='utf-8') as f:
        content_en = f.read()
    
    with open(content_ar_path, 'r', encoding='utf-8') as f:
        content_ar = f.read()
    
    # Connect to database
    conn = sqlite3.connect(db_path)
    cursor = conn.cursor()
    
    # Update English translation
    cursor.execute("""
        UPDATE translations 
        SET content = ? 
        WHERE item_id = 1 AND locale = 'en'
    """, (content_en,))
    
    # Update Arabic translation
    cursor.execute("""
        UPDATE translations 
        SET content = ? 
        WHERE item_id = 1 AND locale = 'ar'
    """, (content_ar,))
    
    conn.commit()
    conn.close()
    
    print("✓ Successfully updated Getting Started content!")
    print("  - English content updated")
    print("  - Arabic content updated")
    
except Exception as e:
    print(f"✗ Error: {e}")
    exit(1)

File: --Legacy/--Docs/update-getting-started.php
<?php

require_once __DIR__ . '/src/Database.php';

use App\Database;

$pdo = Database::getConnection();

// Read content files
$contentEn = file_get_contents(__DIR__ . '/content/getting-started-en.html');
$contentAr = file_get_contents(__DIR__ . '/content/getting-started-ar.html');

// Escape single quotes for SQL
$contentEn = str_replace("'", "''", $contentEn);
$contentAr = str_replace("'", "''", $contentAr);

try {
    $pdo->beginTransaction();

    // Update English translation
    $stmt = $pdo->prepare("
        UPDATE translations 
        SET content = :content 
        WHERE item_id = 1 AND locale = 'en'
    ");
    $stmt->execute([':content' => $contentEn]);

    // Update Arabic translation
    $stmt = $pdo->prepare("
        UPDATE translations 
        SET content = :content 
        WHERE item_id = 1 AND locale = 'ar'
    ");
    $stmt->execute([':content' => $contentAr]);

    $pdo->commit();
    
    echo "✓ Successfully updated Getting Started content!\n";
    echo "  - English content updated\n";
    echo "  - Arabic content updated\n";
    
} catch (\Exception $e) {
    $pdo->rollBack();
    echo "✗ Error: " . $e->getMessage() . "\n";
    exit(1);
}
